src/main/kotlin/io/github/mojira/arisa/modules/TransferLinksModule.kt

```diff
@@ -28,8 +27,8 @@ class TransferLinksModule : AbstractTransferFieldModule() {
     private fun parentDoesNotHaveLink(parentLinks: List<Link>, other: Link) =
         parentLinks.none {
             it.type == other.type &&
-                it.outwards == other.outwards &&
-                it.issue.key == other.issue.key
+                    it.outwards == other.outwards &&
```

Comment:
You have continuation space set to 8?

Reply:
Yes, which I really don't like lol. Changed to 4.